call_16_test.go

```diff
@@ -18,11 +18,11 @@ func TestConnection_Call(t *testing.T) {
 	defer conn.Close()
 
 	// Call16
-	resp, err = conn.Call("simple_incr", []interface{}{1})
+	resp, err = conn.Call("simple_concat", []interface{}{"1"})
```

Comment:
It is not too hard, but it will require additional conversions (`convertUint64` call).

It seems to me clearer to use `string` return type. The returned type is not important in the tests, it is enough to check the value.